src/store/index.js

```diff
@@ -13,10 +13,13 @@ import staticData from './modules/staticData';
 const createStore = () =>
   new Vuex.Store({
     actions: {
-      async nuxtServerInit({ commit }, ctx) {
+      async nuxtServerInit({ commit }, { req }) {
         try {
-          const userInfo = await this.$axios.$get(api.getLoginStatus());
-          commit(types.USER_SET_USER_INFO, userInfo);
+          // eslint-disable-next-line no-underscore-dangle
+          if (req.cookies && req.cookies.__session) {
```

Comment:
Should access by the `AUTH_COOKIE_NAME` constant?

Reply:
skip one comment line too

Reply:
this is in frontend, `AUTH_COOKIE_NAME` was defined in `/server
dont really want to import from server constant, any suggestions?
another `constant` in frontend？